Add Committee 2017 and fix Travis

luketainton opened the pull request:
This branch is now ready to be merged into master (I hope). I've basically cloned the committee section and made a 2017 section directly underneath. Tom, Ollie, and Pandelis all have their new roles along with their photos and links. Bethany does not yet have a photo and as SU Liaison is not yet filled, this is showing as 'PENDING ELECTION'. 

I also removed the arrow's animation to stop the whole site from bouncing up and down.

Also fix Travis CI build errors.

Commit:
Created section for committee 2017

PandelisZ committed:
Im ok without facebook please

Commit:
Changed Ollie's name

Commit:
Changed Ollie's photo

Commit:
Merge remote-tracking branch 'origin/master' into Committee-2017

Commit:
Tidied up Committee 17 section

PandelisZ commented:
I think it's finally time to fix the tests as well. I shall do that in the morning. Looks like misaligned divs and some alt texts to be added to committee images. 

Lets fix #29 in this PR aswell

PandelisZ commented on _data/committee2017.yml:
```diff
@@ -0,0 +1,23 @@
+- name: Tom
```
Maybe reorganise the committee.yml data file? We could have a block for active committee and a block for 2k17 a block for 2k16 etc... Otherwise many data files will exist and thats confusing and unorganised

luketainton replied:
How do you do blocks in that file? I found this in the documentation but it doesn't really help: https://jekyllrb.com/docs/datafiles/

PandelisZ replied:
dunno something to figure out :P consult the yaml jekyll docs

PandelisZ replied:
Indent by one possibly?

commitee.yaml
```
committee-2017
   data

committee-2016
  data

```

and change the jekyll line in index to reflect

PandelisZ commented on index.html:
```diff
@@ -50,6 +52,7 @@
                 <a href="#events"><li>Events <div class="nav-line"></li></a>
                 <a href="#committe"><li>Committee <div class="nav-line"></li></a>
                 <a href="https://www.bcusu.com/organisation/9907/"><li>Membership <div class="nav-line"></li></a>
+                <a href="https://hacshub.luketainton.co.uk/hacshub"><li>HaCSHub <div class="nav-line"></li></a>
```
Imma allow this right now, however in our rethink of the site we need a section/page for member projects to be fair to everyone. Also i need to get you the hacs.tech domain on this

luketainton commented:
I tried that and it wiped out the entire section, I'll check the documentation tomorrow and work on it at some point.

luketainton commented:
@PandelisZ I can't find any way to make that happen except for the way that I've got it set up at the moment :(

Commit:
Fixed 'alt' errors from Travis CI

Commit:
Fixed some alignment errors

PandelisZ committed:
Move all committee data into one file

As discussed cleaned up the data file format by moving all to one file. Also removed old committee from website, will move to seperate page in future

PandelisZ commented on index.html:
```diff
@@ -169,52 +169,12 @@ <h4 class="date">{{event.date}}</h4>
 
             <div class="committee-content">
-              {% for member in site.data.committee2017 %}
+              {% for member in site.data.committee[2017] %}
```
found how to do it @luketainton took some googling but it creeped up in the jekyll docs, I just didnt read them thoroughly enough http://jekyllrb.com/docs/datafiles/

luketainton commented:
Aside from Travis issues, are we ready to push to master?

PandelisZ committed:
fixed all missing divs and ids

PandelisZ reviewed (approved):
Aside from travis he says, fixed.

Merge when you want looks fine to me now

luketainton commented:
@PandelisZ First time I've seen a Travis build pass since introducing it on the repo!

luketainton commented:
All good, merging w/ master.

PandelisZ commented:
It did find legitimate errors! Missing end divs everywhere

luketainton commented:
Once the build on master passes, I'll delete the Committee-2017 branch.
